test/test_helper.js

```diff
@@ -0,0 +1,21 @@
+const request = require('supertest');
+const { app } = require('../src/expressServer.js');
+
+const loginAsAdmin = async (credentials) => {
```

Comment:
We should be able to use this to log in as a `user` role user and as an `admin` user. I don't think there's anything in here that is specific to logging in as an `admin`. It just depends on what credentials we pass into the function with. 

```suggestion
const loginAndReturnCookie = async (credentials) => {
```